SPEC.md

```diff
@@ -621,9 +643,8 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
 ```
 
 * **acVersion** (string, required) represents the version of the schema spec (must be in [semver](http://semver.org/) format)
-* **acKind** (string, required) must be set to "ContainerRuntimeManifest"
-* **uuid** (string, required) must be an [RFC4122 UUID](http://www.ietf.org/rfc/rfc4122.txt) that represents this instance of the container (must be in [RFC4122](http://www.ietf.org/rfc/rfc4122.txt) format)
-* **apps** (list of objects, required) list of apps that will execute inside of this container. Each app object has the following set of key-value pairs:
+* **acKind** (string, required) must be set to "PodManifest"
+* **apps** (list of objects, required) list of apps that will execute inside of this pod. Each app object has the following set of key-value pairs:
     * **app** (string, optional) name of the app (restricted to AC Name formatting)
     * **imageID** (string, required) content hash of the image that this app will execute inside of (must be of the format "type-value", where "type" is "sha512" and value is the hex encoded string of the hash)
     * **mounts** (list of objects, optional) list of mounts mapping an app mountPoint to a volume. Each mount has the following set of key-value pairs:
```

Comment:
@jonboulle yes. So the image might assume that /opt/foo always has contents "foo" because it didn't specify /opt as dest for any volume. And then it runs and finds that /opt/foo contains "bar".